google/cloud/forseti/notifier/notifiers/findings.py

```diff
@@ -29,11 +29,22 @@
 class Findingsnotifier(object):
     """Upload violations to GCS bucket as findings."""
 
+    def __init__(self, inv_index_id):
+        """`Findingsnotifier` initializer.
+
+        Args:
+            inv_index_id (str): inventory index ID
+        """
+        self.inv_index_id = inv_index_id
+
     @staticmethod
-    def _transform_to_findings(violations):
+    def _transform_to_findings(inv_index_id, violations):
```

Comment:
This doesn't need to be a static method anymore, you can use the inv_index_id from self.

Reply:
Done.